docs/pythonpath_support.md

```
@@ -0,0 +1,207 @@
+
+```bash
+PYTHONPATH="/path/to/dir1:/path/to/dir2:/path/to/dir3"
+```
+
+## Examples
+
+### Basic Example
+
+Directory structure:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)